workers/unity/Packages/com.improbable.gdk.core/Tests/Editmode/GameObjectRepresentation/Readers/ReaderWriterTestsBase.cs

```diff
@@ -21,7 +23,8 @@ public void SetUp()
             world = new World("test-world");
             EntityManager = world.GetOrCreateManager<EntityManager>();
             Entity = EntityManager.CreateEntity(typeof(SpatialOSBlittableComponent));
-            Reader = new SpatialOSBlittableComponentReader(Entity, EntityManager, new LoggingDispatcher());
+            ReaderPublic = new BlittableComponent.ReaderWriterImpl(Entity, EntityManager, new LoggingDispatcher());
```

Comment:
Nit: is it possible to assign to ReaderInternal first and then set ReaderPublic = ReaderInternal instead? may be cleaner :)